calico-enterprise/network-policy/application-layer-policies/alp.mdx

```diff
@@ -8,32 +8,39 @@ Application layer policies let you configure access controls based on L7 attribu
 
 ## Before you begin
 
-### Unsupported
-- {{prodname}} implements application layer policy using Envoy as a DaemonSet. This means you cannot use application layer policy alongside a service mesh like Istio.
-- GKE
-
 ### Limitations
-- Application layer policy supports restricting only ingress traffic
-- Support for L7 attributes are limited to HTTP method and URL exact/prefix path 
-- Supported protocols are limited to TCP-based protocols (for example, HTTP, HTTPS, or gRPC)
-- You can control application layer policies only at the cluster level (not per namespace) 
 
-## How to
-- [Enable application layer policies](#enable-application-layer-policies)
-- [Enforce application layer policies for ingress traffic](#enforce-application-layer-policies-for-ingress-traffic)
-- [Disable application layer policies](#disable-application-layer-policies)
+* Application layer policy is not compatible with a service mesh such as Istio.
+* Application layer policy can restrict only _ingress_ traffic.
+  It can not restrict egress traffic.
+* Support for L7 attributes is limited to HTTP method and URL exact/prefix path.
```

Comment:
I'm not sure these two bullets about attributes and protocols are limitations - they're just what the feature currently supports. I know they're pre-existing as limitations so it's fine to leave them I guess, I just think the section is less about what the feature supports / was implemented so far vs. things like platform limitations etc